### DISCUSSION TITLE
Improve release process

### DISCUSSION THREAD

**Discussion by dvaumoron** [tofuutils/tenv#310] [Announcements]
Following the breaking change issue of v4, I think it would be better to improve our release process (particularly in case of major version update related to breaking change). One way could be to release beta versions to allow our growing community to test before an actual release...

Any though ?

**Comment by kvendingoldo** [comment]
@dvaumoron Agree. I will work on proposal this week to add the support of alpha/beta versions.

**Comment by kvendingoldo** [comment]
@dvaumoron I've tuned goreleaser and now we can make tags like v.1.2.3-alpha1 / v.1.2.3-beta2 / v.1.2.3-rc3. This tags won't trigger slack notifications about build, but users will be able to test such versions. Regarding changelog, we're still talking with @Nmishin about it.
